.travis.yml

```diff
@@ -170,13 +170,69 @@ jobs:
         - docker run --rm -it -v "$(pwd)":/mnt kuzzleio/sdk-cross:node8-aarch64 ./.ci/scripts/install-aarch64-deps.sh
         - docker-compose -f .ci/test-aarch64.yml run kuzzle
 
+# ---------------------------------------
+# Node 8 - Redis 4
+# ---------------------------------------
+    - stage: "Tests & Linters"
+      if: (type = push AND branch =~ ^master|.*-stable$) OR type = cron
+      name: "Node 8 - Redis 4"
+
+      os: linux
+      sudo: required
+      language: minimal
+      services: docker
+
+      cache:
+        directories:
+          - $HOME/.pm2 
+
+      env:
+        - DOCKER_COMPOSE_VERSION=1.9.0
+        - NODE_LTS=8
+        - REDIS_VERSION=4.0.14
+        - DOCKER_PROXY_TAG=$(if [ "${TRAVIS_BRANCH}" != "master" ]; then echo ":develop"; fi)
+
+      before_install:
+        - sudo sysctl -w vm.max_map_count=262144
+
+      script:
+        - docker-compose -f .ci/test.yml run kuzzle
+
+# ---------------------------------------
+# Node 8 - Redis 5
+# ---------------------------------------
+    - stage: "Tests & Linters"
+      if: (type = push AND branch =~ ^master|.*-stable$) OR type = cron
+      name: "Node 8 - Redis 5"
+
+      os: linux
+      sudo: required
+      language: minimal
+      services: docker
+
+      cache:
+        directories:
+          - $HOME/.pm2 
+
+      env:
+        - DOCKER_COMPOSE_VERSION=1.9.0
+        - NODE_LTS=8
+        - REDIS_VERSION=latest
```

Comment:
Shouldn't we rather stick to a tag (can be set to `5` imo to follow last versions)?

Reply:
Yes you're right :slightly_smiling_face: